src/functionalTest/java/uk/gov/hmcts/reform/ccd/document/am/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.ccd.document.am.befta;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private Logger logger = LoggerFactory.getLogger(DataStoreTestAutomationAdapter.class);
+
+    private static final String[] TEST_DEFINITIONS_NEEDED_FOR_TA = {
+        "src/functionalTest/resources/CCD_CNP_27.xlsx",
+        "src/functionalTest/resources/CCD_CNP_27_AUTOTEST1.xlsx",
+        "src/functionalTest/resources/CCD_CNP_27_AUTOTEST2.xlsx",
+        "src/functionalTest/resources/CCD_CNP_RDM5118.xlsx",
+
+        "src/functionalTest/resources/CCD_BEFTA_JURISDICTION1.xlsx",
+        "src/functionalTest/resources/CCD_BEFTA_JURISDICTION2.xlsx",
+        "src/functionalTest/resources/CCD_BEFTA_JURISDICTION3.xlsx"
+    };
+
+    private static final String[][] CCD_ROLES_NEEDED_FOR_TA = {
+            { "caseworker-autotest1", "PUBLIC" },
+            { "caseworker-autotest1-private", "PRIVATE" },
+            { "caseworker-autotest1-senior", "RESTRICTED" },
+            { "caseworker-autotest1-solicitor", "PRIVATE" },
+
+            { "caseworker-autotest2", "PUBLIC" },
+            { "caseworker-autotest2-private", "PRIVATE" },
+            { "caseworker-autotest2-senior", "RESTRICTED" },
+            { "caseworker-autotest2-solicitor", "PRIVATE" },
+
+            { "caseworker-befta_jurisdiction_1", "PUBLIC" },
+
+            { "caseworker-befta_jurisdiction_2", "PUBLIC" },
+            { "caseworker-befta_jurisdiction_2-solicitor_1", "PUBLIC" },
+            { "caseworker-befta_jurisdiction_2-solicitor_2", "PUBLIC" },
+            { "caseworker-befta_jurisdiction_2-solicitor_3", "PUBLIC" },
+            { "citizen", "PUBLIC" },
+
+            { "caseworker-befta_jurisdiction_3", "PUBLIC" },
+            { "caseworker-befta_jurisdiction_3-solicitor", "PUBLIC" }
+    };
```

Comment:
Kindly keep only CCD_BEFTA_JURISDICTION2 specific roles for now.

Reply:
Removed other data entries except Jurisdiction2.